pennylane_qiskit/qiskit_device.py

```diff
@@ -59,23 +59,24 @@ def pauli_eigs(n):
     "PauliY": ex.YGate,
     "PauliZ": ex.ZGate,
     "Hadamard": ex.HGate,
-    "CNOT": ex.CnotGate,
-    "CZ": ex.CzGate,
+    "CNOT": ex.CXGate,
```

Comment:
I wonder why they renamed this? CNOT is much more established than CX

Reply:
Really good question. From what I saw they started porting every gate to this type of naming (e.g. `FredkingGate` was also renamed to `CSwapGate`) and I also noticed a base class for controlled X that was created.

They seem to not even mention `CNOT` with the new formalism (once the `CnotGate` has been deprecated, it will not be in there):
https://qiskit.org/documentation/_modules/qiskit/extensions/standard/x.html#CXGate